pkg/storage/rdbms/models/modules.go

```diff
@@ -0,0 +1,60 @@
+package models
+
+import (
+	"encoding/json"
+	"github.com/gobuffalo/pop"
+	"github.com/gobuffalo/uuid"
+	"github.com/gobuffalo/validate"
+	"github.com/gobuffalo/validate/validators"
+	"time"
```

Comment:
nit: can you separate the standard library packages and the others with a newline?